Common/Data/Consolidators/ClassicRenkoConsolidator.cs

```diff
@@ -138,89 +129,40 @@ public ClassicRenkoConsolidator(decimal barSize, RenkoType type)
             PyObject selector,
             PyObject volumeSelector = null,
             bool evenBars = true)
-            : this(barSize, evenBars)
+            : base(selector, volumeSelector)
         {
-            using (Py.GIL())
-            {
-                if (selector != null && !selector.IsNone())
-                {
-                    if (!selector.TryConvertToDelegate(out _selector))
-                    {
-                        throw new ArgumentException(
-                            "Unable to convert parameter 'selector' to delegate type Func<IBaseData, decimal>");
-                    }
-                }
-                else
-                {
-                    _selector = x => x.Value;
-                }
-
-                if (volumeSelector != null && !volumeSelector.IsNone())
-                {
-                    if (!volumeSelector.TryConvertToDelegate(out _volumeSelector))
-                    {
-                        throw new ArgumentException(
-                            "Unable to convert parameter 'volumeSelector' to delegate type Func<IBaseData, decimal>");
-                    }
-                }
-                else
-                {
-                    _volumeSelector = x => 0;
-                }
-            }
+            EpsilonCheck(barSize);
+            _barSize = barSize;
+            _evenBars = evenBars;
```

Comment:
Call `this()` instead of base and we can remove the duplicate body of this constructor

Reply:
Hmm but how that could be done if the other constructor receives selectors of type `Func<IBaseData, decimal>` and these ones are `PyObjects`?

Reply:
You're right 🚀